### DISCUSSION TITLE
What are the login credentials for the demo instance?

### DISCUSSION THREAD

**Discussion by lguenth** [jellyfin/jellyfin#9428] [Q&A] [answered]
Hi! 
Question in title, I wanted to try out https://demo.jellyfin.org/stable but can't find the dummy credentials. :sweat_smile:

**Comment by nielsvanvelzen** [comment, marked as answer]
The username is `demo`, no password. It's shown below the login field:

![image](https://user-images.githubusercontent.com/2305178/222913051-07a64636-ec25-4e30-a6d8-5e5dd699c1be.png)

**Comment by steve-piner** [comment]
Hi, for what it's worth, I also couldn't find the credentials - the message @nielsvanvelzen points out just wasn't showing for me.

I've traced it to a specific extension I have enabled: [I don't care about cookies](https://addons.mozilla.org/en-US/firefox/addon/i-dont-care-about-cookies/), which injects CSS that hides the .disclaimerContainer class wrapping the login message.

Hopefully this helps if anyone is puzzled about how the message might have been missed!

**Reply by lguenth** [reply]
Oh damn, thanks for pointing that out! I really should've checked if something on my end is interfering.

**Reply by nielsvanvelzen** [reply]
Right, I forgot about that but some ad blockers can block the disclaimer element. If I remember correctly this should be fixed in 10.9.

**Reply by jakariyaa** [reply]
Same here! Same thing happened to me today

**Comment by supertin** [comment]
Just a note that this is still a problem. I just get a very blank login screen with standard username/password boxes, and definitely no message about the credentials for the demo.

**Reply by nielsvanvelzen** [reply]
Blame your adblocker/browser/whatever for blocking the element.

**Comment by supertin** [comment]
> Blame your adblocker/browser/whatever for blocking the element.

Why? It doesn't happen on demo sites for other stuff. Shouldn't the login details just be plain text stuck on the page, and not a "disclaimer"? Do you seriously use the internet today without an ad blocker and other addons to make websites actually usable?

**Reply by crobibero** [reply]
I use an adblocker and the disclaimer has never been removed for me. Also noted above, this is fixed for 10.9. You can verify this by visiting https://demo.jellyfin.org/unstable

**Reply by MrChip53** [reply]
I do not use ad blockers and it has never been removed for me either.

**Reply by ferferga** [reply]
This is the DOM:

![image](https://github.com/jellyfin/jellyfin/assets/10274099/f437d7d1-0abb-4892-b9f5-a7b5f9e048c1)

There's nothing shady that sets us apart of other "demo" instances. Your extension just blocked it because it matched the class name we also used.

I use uBlock, which is a reputable good adblocker. There are a lot of adblockers out there and not all of them are good, but nothing we can do about.

**Comment by aaemon** [comment]
How does the demo server prevent users from changing the demo account's password? I want to implement a similar feature on my Jellyfin server.

**Reply by nielsvanvelzen** [reply]
The server is reset at the start of each hour, so if someone changes the password it will be correct again the next hour.
